go.mod

```diff
@@ -4,11 +4,13 @@ module github.com/cosmos/ibc-go/v6
 
 require (
 	cosmossdk.io/math v1.0.0-beta.4
+	cosmossdk.io/simapp v0.0.0-20221208102534-4997976e49ce
+	cosmossdk.io/tools/rosetta v0.2.0
 	github.com/armon/go-metrics v0.4.1
 	github.com/cosmos/cosmos-proto v1.0.0-beta.1
-	// cosmos-sdk main @20448048a489 ≃ 0.47
-	github.com/cosmos/cosmos-sdk v0.47.0-alpha2
+	github.com/cosmos/cosmos-sdk v0.47.0-alpha2.0.20221208102534-4997976e49ce
```

Comment:
Does the rosetta fix require this particular commit or is it possible with official `v0.47.0-alpha2` release tag? 

I appreciate keeping up to date with `main`, my only concern is more volatility which could lead to fixing and re-fixing before the sdk team is ready to officially tag a release.